package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "airport-node-module",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
This needs a start script -> `"start": "node ./servers/server.js"`
Else the `server.js` defined at root will be taken during npm start